File: onebot-client/src/main/java/cn/evole/onebot/sdk/response/contact/FriendInfoResp.java
package cn.evole.onebot.sdk.response.contact;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class FriendInfoResp {

    @SerializedName("user_id")
    private long userId;

    @SerializedName("nickname")
    private String nickname;

    @SerializedName("remark")
    private String remark;

}

File: overflow-core/src/main/kotlin/cn/evolvefield/onebot/sdk/response/contact/FriendInfoResp.kt
package cn.evolvefield.onebot.sdk.response.contact

import com.google.gson.annotations.SerializedName

class FriendInfoResp  {
    @SerializedName("user_id")
    var userId = 0L
    @SerializedName(value = "nickname", alternate = ["user_name"])
    var nickname = ""
    @SerializedName(value = "remark", alternate = ["user_remark"])
    var remark = ""
    @SerializedName("sex")
    var sex = "unknown"
    @SerializedName("age")
    var age = 0
    @SerializedName("email")
    var email = ""
    @SerializedName("level")
    var level = 0
}
